style.css

```diff
@@ -1 +1,194 @@
 /*  Add your styling here */
+body{
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    display: grid;
+    grid-template-areas: 
+    "header"
+    "main"
+    "main2"
+    "footer";
+
+}
+a {
+    color: rgb(122, 0, 122);
+}
+
+
+.header1{
+    grid-area: header;
+    background-color: rgb(235, 195, 123);
+    position: fixed;
+    box-shadow: 1px 1px 4px rgba(0,0,0,.2);
+    width: 100%;
+    z-index: 10;
+    height:75px;
+}
+
+.header1 ul{
+    list-style: none; 
+    overflow: hidden;
+}
+
+.header1 ul a{
+    display: block;
+    /* outline: 1px solid purple; */
+    padding:15px;
+    border:1px solid purple;
+    text-decoration: none;
+}
+
+.header1 .logo{
+    float: left;
+    display: block;
+    margin-top: 5px;
+    margin-left: 5px;
+}
+.header1 ul a:hover{
+    background-color:rgb(226, 165, 52);
+    text-decoration: none;
+}
+.header1 .menu{
+    clear: both;
+    max-height: 0;
+    transition: max-height .2s ease-out;
+}
+.header1 .menu-icon{
+    /* border: 1px solid purple; */
+    padding: 28px 20px;
+    position: relative;
+    float: right;
+    cursor: pointer;
+}
+.header1 .menu-icon .nav-icon{
+    background: black;
+    display: block;
+    height: 2px;
+    width: 18px;
+    position: relative;
+    transition: background .2s ease-out;
+}
+.header1 .menu-icon .nav-icon:before{
+    background: black;
+    content:"";
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    top: 5px;
+    
+}
+.header1 .menu-icon .nav-icon:after{
+    background: black;
+    content:"";
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    top: -5px;
+}
+.header1 .menu-btn{
+    display: none;
+}
+.header1 .menu-btn:checked ~ .menu{
+    background-color: rgb(235, 195, 123);
+    max-height: 300px;
+    margin-top: 80px;
+    margin-right: 5%;
+    margin-left: 50%;
+}
+
+.header1 .menu-btn:checked ~ .menu-icon .nav-icon{
+    background-color:transparent;
+
+}
+.header1 .menu-btn:checked ~ .menu-icon .nav-icon::before{
+    transform: rotate(45deg);
+    top:0;
+}
+.header1 .menu-btn:checked ~ .menu-icon .nav-icon:after{
+    transform: rotate(-45deg);
+    top:0;
+}
+.aside1{
+    color: rgb(83, 7, 83);
+}
+
+.main1{
+    grid-area: main;
+    background-color: rgb(255, 185, 56);
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    margin-top: 75px;
+    grid-column-gap: 5px;
+}
+.footer1{
+    grid-area: footer;
+    border:1px solid black;
+    background-color: grey;
+    text-align: center;
+}
+.section1{
+    color: purple;
+    text-align: center;
+}
+.section2{
+    grid-area: main2;
+    float: left;
+}
+.twitter{
+    width: 30px;
+    margin-right: 1em;
+}
+figcaption{
+    text-align: center;
+    text-shadow: saddlebrown;
+}
+p{
+    font-size: medium;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+@media (min-width:768px){
+    .header1 li{
+        float: left;
```

Comment:
Do't use floats please - use flexbox instead

Reply:
it didn't work with : display: flex; flex-direction:row;
it still stay in column, I changed it anyway and make float a comment.